www/src/data/sidebars/doc-links.yaml

```diff
@@ -259,6 +259,11 @@
           link: /docs/seo/
         - title: Optimizing Site Performance with Guess.js
           link: /docs/optimizing-site-performance-with-guessjs/
+    - title: Localizing your site
+      link: /docs/localization-i18n/
```

Comment:
It should! Sounds like the scope of a different PR though.

Reply:
Well - not sure! This would introduce a broken link, no?

A convention we’ve followed elsewhere is to create a stub for content that isn’t fleshed out yet—but should be created.

Reply:
Ah, yes! A stub would be great in the meantime. What's the process for creating those?

Reply:
@marcysutton here's an example PR that implements that pattern:

https://github.com/gatsbyjs/gatsby/pull/11423/files